### PR TITLE
Add dart support

### PR DESCRIPTION
**Pull request by andremeireles**
This adds support for the dart language, based on the lua implementation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 
+## [1.5.0]
```
Comment (chlohal) [inline CHANGELOG.md:8]: In general, @twibiral is the one to update the changelog when it is time for the next release. I'm not sure what the correct procedure is or what he would prefer re: this commit, but I wanted to make sure you were aware of that :) sorry!

**Review by chlohal — COMMENTED** [review]
+1 Your code looks great! One note regarding the changelog, but everything else looks nice. Thank you for your contribution!

File: CHANGELOG.md (hunk 1 above)
Comment (twibiral) [reply CHANGELOG.md:8]: @chlohal is right, I would prefer if you could remove this @andremeireles. The version number is finalized before a new version is released and added to the CHANGELOG by a script

**Comment by twibiral** [comment]
Thank you for the contribution @andremeireles, I really appreciate it!

Before merging, please do the change commented above and resolve the merge conflicts (this should be pretty straightforward)

File: CHANGELOG.md (hunk 1 above)
Comment (andremeireles) [reply CHANGELOG.md]: Ah, great.
I think it is now ready to receive the merge.

Comment (chlohal) [reply CHANGELOG.md]: LGTM!

**Review by chlohal — APPROVED** [review]

**Comment by commonCereal** [comment]
Omg I'm excited for this, just discovered this plugin and use Dart professionally, nice job @andremeireles !!

**Comment by andremeireles** [comment]
> Omg I'm excited for this, just discovered this plugin and use Dart professionally, nice job @andremeireles !!

I don’t know what to say. Thank you.

**Comment by commonCereal** [comment]
@twibiral  is there anything else needed for this to go through? No rush of course, just curious ❤️

**Comment by twibiral** [comment]
It will be released soon :)

**Pull request merged**